### PR TITLE
Fix group spec

### PR DESCRIPTION
**Pull request by iamjarvo**
Depends on docker branch being merged. Will rebase once its merged

### CODE REVIEW

File: spec/group_spec.rb [hunk 1]
```diff
@@ -9,11 +9,25 @@
...
+  let(:first_group) do
+    Vinyldns::API::Group.list_my_groups["groups"].find do |group|
+      group["name"] == "test-group"
+    end
+  end
 
   describe '.create' do
     it 'can POST & receives 409 Conflict connecting to an already existing group' do
-      expect(Vinyldns::API::Group.create(first_group['name'], first_group['email'], first_group['members'], first_group['admins']).class.name).to eq('Net::HTTPConflict')
+      expect(Vinyldns::API::Group.create(first_group['name'], first_group['email'], [], [], 'description').class.name).to eq('Net::HTTPConflict')
```
Comment (britneywright) [inline spec/group_spec.rb:30]: you don't need to include description here, it can be an empty string

Comment (iamjarvo) [reply spec/group_spec.rb:30]: vinyl requires the string to not be empty. I should update the method to not default to `''` wanted to make minimal changes to the code though

Comment (iamjarvo) [reply spec/group_spec.rb:30]: The fix for this is to have the client not send a description if the description field == ''

Comment (britneywright) [reply spec/group_spec.rb:30]: https://github.com/vinyldns/vinyldns-ruby/issues/12

**Pull request merged**